homeassistant/components/wevolor/__init__.py

```diff
@@ -0,0 +1,35 @@
+"""The Wevolor Control for Levolor Motorized Blinds integration."""
+from __future__ import annotations
+
+from pywevolor import Wevolor
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN
+
+PLATFORMS: list[str] = ["cover", "button"]
```

Comment:
Limit the PR to a single platform. You can create a Pull Requests for the other platform when this merges.
https://developers.home-assistant.io/docs/creating_component_code_review/#5-make-your-pull-request-as-small-as-possible

Reply:
Got it. Removed the button for now.